extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/runtime/MongodbConfig.java

```diff
@@ -14,6 +14,14 @@ public class MongodbConfig {
 
     public static final String CONFIG_NAME = "mongodb";
 
+    public static final String NATIVE_DNS_LOG_ACTIVITY = "native.dns.log-activity";
```

Comment:
Isn't the new resolver used also in JVM mode?

Reply:
You can introduce a new property and alias that one. @radcortez explained to me that a long time ago.

Reply:
You can use a Relocate: https://smallrye.io/smallrye-config/2.10.2-SNAPSHOT/extensions/relocate/